assets/feature-installer-utils.sh

```diff
@@ -1,4 +1,31 @@
-# Execute a command and if it fails, return its output and exit
+# Set some informative variables
+
+# The flavour we're running on
+FLAVOUR="$(cat /home/cloudcontrol/flavour)"
+export FLAVOUR
+
+# The path to install software binaries to
+BINPATH="/home/cloudcontrol/bin"
+export BINPATH
+
+TEMPDIR=""
+export TEMPDIR
+
+# Prepare feature installation. Will create a temporary directory and change to it
+function prepare {
```

Comment:
If a function is just doing one thing, you should name it that way.
Here `createTempDir` would match.

Reply:
I disagree tbh. This function also moves into the temporary dir and from the feature installer script, this both is used to prepare a workspace for it.

This function currently is only used in the template, but should be rolled out to the other features as well, but I didn't want to blow up the PR.